2024/survey.yaml

```diff
@@ -356,6 +356,19 @@ questions:
       - Wiki (unofficial)
       - Blog posts
       - Others
+  - prompt: If you want to contribute or have contributed in the past to Nixpkgs, please select all that apply:
+    type: multiple
+    choices:
+      - I don't plan to contribute.
+      - Everything I want to contribute to is already being worked on and I don't feel the need to get involved.
+      - I would like to contribute but I don't know how to get started.
+      - My experience contributing was as expected.
```

Comment:
What do you mean by that? It feels like other choices provide insight into this choice with more detail already.

Reply:
This was meant to be a neutral answer. We can remove if it is superfluous.

Reply:
```suggestion
```